### ISSUE TITLE
Builds screen UI should update to show new builds that have been triggered

### ISSUE THREAD

**Issue by andrewedstrom** [concourse/concourse#1227]
# Feature Request

## What challenge are you facing?

When I am on the Builds screen for a particular job, I am not able to see builds that have started since I first loaded the page.

For example, in ci.concourse.ci, if I have the `atc` Builds screen open in two tabs A and B, when I trigger a new build in tab A, I am not able to see that this new build has started in the top bar of tab B.

## A Modest Proposal

The list of build ids in the top of the Builds screen should automatically refresh when new builds are triggered.

**Comment by marco-m** [comment]
Same problem (web UI not updated) when triggering with `fly`.

**Comment by timrchavez** [comment]
We've experienced situations where the job will stay yellow, even though it's completed. The expectation is that it goes green. Sometimes users will wait a really long time for it to transition from yellow to green and _think_ the build is taking a lot longer than it actual has. I'm wondering if this is related?

**Comment by andrewedstrom** [comment]
FWIW I would still love to see this fixed

**Comment by andrewedstrom** [comment]
This really seems like something that should be done still.

**Comment by vito** [comment]
@andrewedstrom yep, we just don't have bandwidth for this ourselves. added 'help wanted' label to keep it alive!

**Comment by taylorsilva** [comment]
# Issue Bankruptcy

This issue is being closed as part of 2025's **Issue Bankruptcy** declaration. [See here for details](https://blog.concourse-ci.org/posts/2025-01-06-concourse-2024-review/)

If you think this issue still has merit then feel free to open a new issue and reference this one.
